### ISSUE TITLE
VTK build fails because it creates an unexpected bin directory.

### ISSUE THREAD

**Issue by FrankHeimes** [Microsoft/vcpkg#1488]
VTK has always created a bin directory to store some additional tools that might be needed in particular cases (i.e. H5detect.exe, H5make_libsettings.exe, lproj.exe, vtkEncodeString*.exe, vtkHashSource*.exe, vtkParseOGLExt*.exe).
It seems that the bin folders to store these tools are created even though the tools themselves are not built.
The vcpkg build complains about these orphan bin folders and aborts the build:
```
vcpkg install vtk:x86-windows-static
The following packages will be built and installed:
    vtk:x86-windows-static
Building package vtk:x86-windows-static... 
-- Package x86-windows-static-dbg done
-- Performing post-build validation
There should be no bin\ directory in a static build, but D:\GitHub\vcpkg\packages\vtk_x86-windows-static\bin is present.
There should be no debug\bin\ directory in a static build, but D:\GitHub\vcpkg\packages\vtk_x86-windows-static\debug\bin is present.
If the creation of bin\ and/or debug\bin\ cannot be disabled, use this in the portfile to remove them

    if(VCPKG_LIBRARY_LINKAGE STREQUAL static)
        file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/bin ${CURRENT_PACKAGES_DIR}/debug/bin)
    endif()

There should be no empty directories in D:/GitHub/vcpkg/packages/vtk_x86-windows-static
The following empty directories were found: 

    D:/GitHub/vcpkg/packages/vtk_x86-windows-static/bin
    D:/GitHub/vcpkg/packages/vtk_x86-windows-static/debug/bin

If a directory should be populated but is not, this might indicate an error in the portfile.
If the directories are not needed and their creation cannot be disabled, use something like this in the portfile to remove them:

    file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/a/dir ${CURRENT_PACKAGES_DIR}/some/other/dir)
```
So apparently, the portfile needs to be adjusted.

**Comment by albertziegenhagel** [comment]
This should be fixed by the PR #1601, although untested since it is blocked by #1478

**Comment by alexkaratarakis** [comment]
#1601 is now merged.
